### PR TITLE
docs: add deleteOldBuildsCron

### PR DESCRIPTION
**Pull request by koh110**
This PR is adding document of deleteOldBuildsCron

Ref: https://github.com/GoogleChrome/lighthouse-ci/pull/471#issuecomment-731609275

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -970,6 +970,18 @@ _Optional_ The human friendly label for this set of URLs to use when logging sta
 
 _Optional_ The "branch" on which to report the results. Defaults to the base branch of the project referenced by `projectSlug`.
 
+#### deleteOldBuildsCron
```
Comment (patrickhulce) [inline docs/configuration.md:973]:
```suggestion
#### `deleteOldBuildsCron`
```

File: docs/configuration.md [hunk 2]
```diff
@@ -970,6 +970,18 @@ _Optional_ The human friendly label for this set of URLs to use when logging sta
...
+
+The configuration to automatically delete old records. This job only works when `storage.storageMethod` is set `sql`.
+
+##### schedule
```
Comment (patrickhulce) [inline docs/configuration.md:977]:
```suggestion
##### `deleteOldBuildsCron.schedule`
```

File: docs/configuration.md [hunk 3]
```diff
@@ -970,6 +970,18 @@ _Optional_ The human friendly label for this set of URLs to use when logging sta
...
+
+The [cron-style](https://crontab.guru/) schedule on which to collect results.
+
+##### maxAgeInDays
```
Comment (patrickhulce) [inline docs/configuration.md:981]:
```suggestion
##### `deleteOldBuildsCron.maxAgeInDays`
```

**Review by patrickhulce — APPROVED** [review]
thanks @koh110! :)

**Comment by patrickhulce** [comment]
@koh110 are you able to rebase this against master now that I've fixed the CI issues? we can land after that :)

**Comment by koh110** [comment]
@patrickhulce Thanks! I rebaseed and pushed!

**Comment by patrickhulce** [comment]
@googlebot I consent.

**Pull request merged**